backend/app/server/models/shop.py

```diff
@@ -1,29 +1,37 @@
 from datetime import datetime, timedelta
-from server.utils.distance import distance
-from beanie import Document
-from pydantic import BaseModel
+from typing import List, Optional
+from pydantic import BaseModel, Field
+from beanie import Document, PydanticObjectId
+
+
+
+class Point(BaseModel):
+    type: str = Field("Point", const=True)
+    coordinates: List[float]
```

Comment:
Żeby było super ładnie zrobiłbym taki alias na typ:
python```
type Coordinates = tuple[float,float] 
```
albo List jak trzeba, aby było jeszcze bardziej czytelne

Reply:
Jest jeszcze jeden problem. Jak zwracamy taki model z tym `Point` to json ma taki fragment:
`    "location": {
      "type": "Point",
      "coordinates": [
        51.1867762716661,
        20.01512298583985
      ]
    },
`
i to jest takie średnie aby to parsować na froncie. Ja też poszukam, ale super byłoby jakby sie dało to zwracać najlepiej bez tego typu.